apps/trading/components/market-selector/asset-dropdown.tsx

```diff
@@ -66,7 +66,7 @@ const TriggerText = ({
 
   if (checkedAssets.length === 1) {
     const assetId = checkedAssets[0];
-    const asset = assets.find((a) => a.id === assetId);
+    const asset = assets.find((a) => a?.id === assetId);
```

Comment:
Why do we need this? If we do, our types are lying to us and there is a problem elsewhere